papermill/abs.py

```diff
@@ -1,10 +1,27 @@
+"""Utilities for working with Azure blob storage"""
 import re
 import io
+import logging
+
 from six.moves import urllib
+
 from azure.storage.blob import BlockBlobService
 
 
+logger = logging.getLogger('papermill.abs')
```

Comment:
Since we don't use this logger anywhere should this still be here?

Reply:
We can remove this for now. We probably should better logging going forward. When we do I would `logger = logging.getLogger(__name__)`

Reply:
Removed